android/app/src/main/java/com/votlin/android/extensions/ViewExtensions.kt

```diff
@@ -0,0 +1,25 @@
+package com.votlin.android.extensions
+
+import android.view.View
+import android.widget.ImageView
+import com.bumptech.glide.Glide
+
+/**
+ * ViewExtensions.
+ */
+fun ImageView.load(url: String) {
+    Glide.with(this.context)
```

Comment:
Glide works with ImageView as with() parameter, we don't need the context :)

Reply:
Nope, our current version of Glide doesn't have with() :(